tensorflow_datasets/image/GroceryStoreDataset_test.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import GroceryStoreDataset
+
+num_classes=len(GroceryStoreDataset._CLASS_NAMES)
+
+class GroceryStore(tfds_test.DatasetBuilderTestCase):
```

Comment:
<code>NameError: name 'tfds_test' is not defined</code>   --- kokoro error (line 694 [here](https://source.cloud.google.com/results/invocations/e4dcd442-8c17-4a97-9656-568f6eac33db/targets/tensorflow_datasets%2Fgh_testing%2Fsubjobs%2Fpy2_tf12/log))
Replace it with <code>testing</code>